apps/dashboard/src/@/components/ui/button.tsx

```diff
@@ -46,6 +46,7 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     const Comp = asChild ? Slot : "button";
     return (
       <Comp
+        type="button"
```

Comment:
this will be weird if someone uses `<Button as=<other component>` that doesn't support `type` (lots of react warnings ahead)

it's ok as long as u first check `Comp === 'button'` (see line 46)

Reply:
Good catch, added condition